### PR TITLE
Milestone: Add styling

### PR DESCRIPTION
**Pull request by amon-cofie**
### Changes:

- Added Book.module.css to add styles to the Book.js component
- Added BooksList.module.css to add styles to the BooksList.js component
- Added FormInput.module.css to add styles to the FormInput.js component
- Added styles to the homepage in the index.css component

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,26 +1,77 @@
...
+    <li className={style.newBook}>
+      <div>
+        <p className={`${style.bookCategory} ${style.commonFontStyles}`}>
+          {book.category}
+        </p>
+        <p className={`${style.bookTitle} ${style.commonFontStyles}`}>
+          {book.title}
+        </p>
+        <p className={`${style.bookAuthor} ${style.commonFontStyles}`}>
+          {book.author}
```
Comment (thecodechaser) [inline src/components/Book.js:21]: - [ ] Please make sure to add the progress bar on books as on the Zeplin design.

### Your design
image

### Zeplin design
image

File: src/components/Book.js [hunk 2]
```diff
@@ -1,26 +1,77 @@
...
+          >
+            CURRENT CHAPTER
+          </p>
+          <p className={`${style.currentChapter} ${style.commonFontStyles}`}>
+            Chapter 4
+          </p>
+          <button className={style.updateProgressBtn} type="button">
+            UPDATE PROGRESS
+          </button>
+        </div>
```
Comment (thecodechaser) [inline src/components/Book.js:73]: - [ ] Please remove the black border around the buttons to match the Zeplin design.

### Your design
image

### Zeplin design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @amon-cofie👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (amon-cofie) [reply src/components/Book.js:21]: I will implement this immediately. Thank you.

File: src/components/Book.js (hunk 2 above)
Comment (amon-cofie) [reply src/components/Book.js:73]: Thank you for pointing that out. I will make the changes immediately

**Review by topeogunleye — APPROVED** [review]
### Hi @amon-cofie, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**